apps/sms/js/thread_ui.js

```diff
@@ -662,50 +660,22 @@ var ThreadUI = {
     }
   },
 
-  handleForward: function thui_handleForward(forward) {
-    var request = MessageManager.getMessage(+forward.messageId);
-
-    request.onsuccess = (function() {
-      Compose.fromMessage(request.result);
-
-      Recipients.View.isFocusable = true;
-      ThreadUI.recipients.focus();
```

Comment:
Mmm, I'll recheck before review request, but as far as I can tell we focus recipients in ```handleActivity``` as well if number is not passed and it's true for forward case